commons/src/main/java/org/apache/mesos/elasticsearch/common/zookeeper/ZookeeperCLIParameter.java

```diff
@@ -20,4 +20,12 @@ public String getZookeeperUrl() {
     public long getZookeeperTimeout() {
         return zookeeperTimeout;
     }
+
+    public static final String ZOOKEEPER_FRAMEWORK_URL = "--zookeeperFrameworkUrl";
```

Comment:
I suggest that if zookeeperFrameworkUrl is not set it gets the value from zookeeperMesosUrl.